build_android.py

```diff
@@ -104,49 +105,48 @@ def set_default_config(file_name):
     if not os.path.exists("build"):
         os.mkdir("build")
 
-    print "Checking dependencies..."
+    print("Checking dependencies...")
     install_dependencies()
 
-    print "Checking for rapt..."
+    print("Checking for rapt...")
     if not os.path.exists("build/rapt"):
         download_rapt()
         unzip_rapt()
         os.unlink("build/" + RAPT_ARCHIVE)
 
-    print "Verifying rapt builder..."
+    print("Verifying rapt builder...")
     modify_rapt()
 
     if not os.path.exists("build/rapt/tuxemon"):
-        print "  Creating symlink to project directory..."
+        print("  Creating symlink to project directory...")
         os.symlink("../../tuxemon", "build/rapt/tuxemon")
 
     # Check md5 hash instead of copying every time.
-    print "Copying icon and splash images..."
+    print("Copying icon and splash images...")
     shutil.copyfile("tuxemon/resources/gfx/icon.png", "build/rapt/templates/pygame-icon.png")
     #shutil.copyfile("tuxemon/resources/gfx/presplash.jpg", "build/rapt/templates/pygame-presplash.jpg")
 
     # Set default config for Android devices.
     set_default_config("tuxemon/tuxemon.cfg")
 
-    print "Executing build..."
+    print("Executing build...")
     os.chdir("build/rapt")
 
-    print "Checking for existing Android SDK..."
+    print("Checking for existing Android SDK...")
     sdk = False
     for item in os.listdir('.'):
         if 'android-sdk-' in item:
             sdk = True
 
     if not sdk:
-        print "  SDK not found."
+        print("  SDK not found.")
         sys.argv.append('installsdk')
-        execfile('android.py')
+        exec('android.py')
```

Comment:
I believe that python3 expects the parameter to for exec to be a string.  In this case, this will not work by passing a filename; the contents of the file must be read into memory first.  We still support python 2.7, so any changes must work with both versions